src/Framework/Framework/Routing/UrlHelper.cs

```diff
@@ -30,7 +31,7 @@ public static string BuildUrlSuffix(string? urlSuffix, object? query)
                     }
                     break;
                 case IEnumerable<KeyValuePair<string, object>> keyValueCollection:
-                    foreach (var item in keyValueCollection)
+                    foreach (var item in keyValueCollection.Where(i => i.Value != null))
```

Comment:
:+1: I guess we should do the same for the other two cases

Reply:
This will be resolved in #1502